Sources/MessagingInApp/Gist/EngineWeb/EngineWeb.swift

```diff
@@ -21,6 +21,8 @@ protocol EngineWebInstance: AutoMockable {
 
 public class EngineWeb: NSObject, EngineWebInstance {
     private let logger: Logger = DIGraphShared.shared.logger
+    private let inAppMessageManager: InAppMessageManager = DIGraphShared.shared.inAppMessageManager
+    private let _currentMessage: Message?
```

Comment:
Can this be part of the constructor? The usage of EngineWeb only seems to be from MessageManager and it get initialized with Message.

Reply:
Updated and made it non-null